examples/basic/main.tf

```diff
@@ -15,14 +15,16 @@ module "resource_group" {
 ##############################################################################
 
 module "cos" {
-  source                 = "../../"
-  resource_group_id      = module.resource_group.resource_group_id
-  region                 = var.region
-  cos_instance_name      = "${var.prefix}-cos"
-  cos_tags               = var.resource_tags
-  bucket_name            = "${var.prefix}-bucket"
-  retention_enabled      = false # disable retention for test environments - enable for stage/prod
-  kms_encryption_enabled = false
+  source                             = "../../"
+  resource_group_id                  = module.resource_group.resource_group_id
+  region                             = var.region
+  cos_instance_name                  = "${var.prefix}-cos"
+  cos_tags                           = var.resource_tags
+  bucket_name                        = "${var.prefix}-bucket"
+  retention_enabled                  = false # disable retention for test environments - enable for stage/prod
+  kms_encryption_enabled             = false
+  activity_tracker_read_data_events  = false # disable activity_tracker
+  activity_tracker_write_data_events = false # disable activity_tracker
```

Comment:
There is no need to set these to false - just remove them and allow the default to be used